### PR TITLE
doc: add README.md

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

Update README.md for new Cyclone.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ddysher** [comment]
coveralls is red, can we fix it?

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+	<br>
+</h1>
+
+Cyclone is a cloud native workflow engine run in Kubernetes. It's implemented with Kubernetes CRD (Custom Resource Definition), with no extra database components required. Once deployed, you will get one cloud-scale workflow engine ready for your workflows.
+
+Cyclone defines each workflow stage as a pod, and assemble them to workflow by capturing dependencies between stages as DAG. 
+
+## Features
+- **Kubernetes Native**: Cyclone run on any Kubernetes clusters without any other dependencies.
+- **CI/CD**:  Cyclone supports CI/CD pipeline natively by templates.
```
Comment (supereagle) [inline README.md]: **Workflow Engine**: Supports CI/CD pipelines, AI pipelines and any other general workflows.

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+	<br>
+	<br>
+</h1>
+
+Cyclone is a cloud native workflow engine run in Kubernetes. It's implemented with Kubernetes CRD (Custom Resource Definition), with no extra database components required. Once deployed, you will get one cloud-scale workflow engine ready for your workflows.
+
+Cyclone defines each workflow stage as a pod, and assemble them to workflow by capturing dependencies between stages as DAG. 
+
+## Features
+- **Kubernetes Native**: Cyclone run on any Kubernetes clusters without any other dependencies.
```
Comment (supereagle) [inline README.md]: Totally based on Kubernetes without any other dependencies.

File: README.md [hunk 3]
```diff
@@ -1 +1,49 @@
...
+- **CI/CD**:  Cyclone supports CI/CD pipeline natively by templates.
+- **AI Workflow**: ...
+- **DAG**: Cyclone supports any DAG workflow.
+- **Resources**: Abstract input/output of workflow as resource and support any resource types as long as resource resolver image provided. For example, SCM, Docker Registry, S3.
+- **Artifacts**: Data generated by one stage can be used by subsequent stages as artifacts.
+- **Triggers**: Workflow can be triggered by time schedule, webhooks.
+- **Controllable**: Execution of workflow can be paused, resumed or cancelled.
+- **Multi-Tenant Support**: Cyclone supports multi-tenant.
```
Comment (supereagle) [inline README.md]: **Multi-Tenant**: Cyclone supports multi-tenant.

File: README.md [hunk 4]
```diff
@@ -1 +1,49 @@
...
+- **Timeout**: Set execution timeout for workflow.
```
Comment (supereagle) [inline README.md]: Minor feature can be ignored.

File: README.md [hunk 5]
```diff
@@ -1 +1,49 @@
...
+- **Garbadge Collection**: Cleanup after workflow finished and results collected.
```
Comment (supereagle) [inline README.md]: **Garbadge Collection**: Cleanups after workflow finished and collects results.

File: README.md [hunk 6]
```diff
@@ -1 +1,49 @@
...
+
+Cyclone defines each workflow stage as a pod, and assemble them to workflow by capturing dependencies between stages as DAG. 
+
+## Features
+- **Kubernetes Native**: Cyclone run on any Kubernetes clusters without any other dependencies.
+- **CI/CD**:  Cyclone supports CI/CD pipeline natively by templates.
+- **AI Workflow**: ...
+- **DAG**: Cyclone supports any DAG workflow.
+- **Resources**: Abstract input/output of workflow as resource and support any resource types as long as resource resolver image provided. For example, SCM, Docker Registry, S3.
+- **Artifacts**: Data generated by one stage can be used by subsequent stages as artifacts.
```
Comment (supereagle) [inline README.md]: Minor feature can be ignored.

File: README.md [hunk 7]
```diff
@@ -1 +1,49 @@
...
+Cyclone is a cloud native workflow engine run in Kubernetes. It's implemented with Kubernetes CRD (Custom Resource Definition), with no extra database components required. Once deployed, you will get one cloud-scale workflow engine ready for your workflows.
+
+Cyclone defines each workflow stage as a pod, and assemble them to workflow by capturing dependencies between stages as DAG. 
+
+## Features
+- **Kubernetes Native**: Cyclone run on any Kubernetes clusters without any other dependencies.
+- **CI/CD**:  Cyclone supports CI/CD pipeline natively by templates.
+- **AI Workflow**: ...
+- **DAG**: Cyclone supports any DAG workflow.
+- **Resources**: Abstract input/output of workflow as resource and support any resource types as long as resource resolver image provided. For example, SCM, Docker Registry, S3.
```
Comment (supereagle) [inline README.md]: Resource is the implementation, maybe can not be understood by readers.
What about:

**Pluggability**: Cyclone can easily integrate with external systems, such as SCM, Docker Registry, S3, etc.

**Pull request merged**